maint.php

```diff
@@ -1051,7 +1275,7 @@ function clearFilter() {
 			form_selectable_cell(number_format_i18n($host['data_sources']), $host['id'], '', 'text-align:right');
 			form_selectable_cell(number_format_i18n($host['tholds']), $host['id'], '', 'text-align:right');
 			form_selectable_cell(get_colored_device_status(($host['disabled'] == 'on' ? true : false), $host['status']), $host['id'], '', 'text-align:center');
-			form_selectable_cell((strlen(get_request_var('filter')) ? preg_replace('/(' . preg_quote(get_request_var('filter')) . ')/i', "<span class='filteredValue'>\\1</span>", htmlspecialchars($host['hostname'])) : htmlspecialchars($host['hostname'])), $host['id']);
+			form_selectable_cell(filter_value($host['hostname'], get_request_var('filter')), $host['id']);
```

Comment:
Not sure why you changed this?  It was designed to be able to highlight the partial matches of text.

Reply:
function.php filter_value() seems to do the magic to highlight the selection.  I copied it from host.php to get the highlighting.  I noticed I missed changing it on the description value and the webseer tab.  Let me know if I misunderstood.